Assets/MRTK/Providers/LeapMotion/Editor/ConfigurationChecker/Microsoft.MixedReality.Toolkit.Providers.LeapMotion.ConfigurationChecker.asmdef

```diff
@@ -1,16 +0,0 @@
-{
-    "name": "Microsoft.MixedReality.Toolkit.Providers.LeapMotion.ConfigurationChecker",
```

Comment:
Is this removal combining the config checker and editor assemblies for Leap Motion?

Reply:
Ah I see that the config checkers were all moved to the editor assembly. Seems fine to me! I think they were originally separate to try to help these scripts load even if a different assembly was broken, but it seems like that doesn't happen anyway (and editor assemblies _should_ be mostly package agnostic at the moment).

Reply:
@CDiaz-MS FYI